packages/vitest/src/node/stdin.ts

```diff
@@ -119,6 +119,18 @@ export function registerConsoleShortcuts(
       return inputFilePattern()
     }
     if (name === 'b') {
+      await Promise.all(ctx.projects.map(async (project) => {
+        if (project.browser) {
+          ctx._browserSessions.sessionIds.clear()
+          project.browser.state.orchestrators.forEach((orchestrator) => {
```

Comment:
I don't think all of this cleanup is required? The WS cache is removed when the connection is severed

`cleanupTesters` is also async, it can call orchestrator again

Even if some state cleanup is required, it shouldn't be done here, it should be part of `browser.close`